pkg/metrics/prometheus.go

```diff
@@ -4,11 +4,28 @@ import (
 	"strings"
 
 	"github.com/prometheus/client_golang/prometheus"
+	"ktbs.dev/teler/common"
 )
 
 // PrometheusInsert logs into metrics
-func PrometheusInsert(data map[string]string) {
+func PrometheusInsert(options *common.Options, data map[string]string) {
 	var counter prometheus.Counter
+	cfg := options.Configs
+
+	rules := make(map[string]bool)
+	for _, custom := range cfg.Rules.Threat.Customs {
+		rules[custom.Name] = true
+	}
+
+	switch {
+	case rules[data["category"]]:
+		counter = getCustomsRule.WithLabelValues(
+			data["category"],
+			data["element"],
+			data[data["element"]],
+		)
+		counter.Inc()
+	}
```

Comment:
Moving this `case` to `switch` below should be fine, and no need to trigger `counter.Inc()`.